sklearn/tree/tests/test_tree.py

```diff
@@ -501,73 +501,6 @@ def test_importances_gini_equal_squared_error():
     assert_array_equal(clf.tree_.n_node_samples, reg.tree_.n_node_samples)
 
 
-def test_max_features():
```

Comment:
It looks like `test_max_features` got moved down the file, but half the code is the same.  Instead of moving the whole `test_max_features` function over, can we delete the `with pytest.raises(ValueError):` parts from this test and keep everything else in place? By keeping the code in place, it makes it easier to run `git blame` and track history.

In other words, keep the following lines:

https://github.com/scikit-learn/scikit-learn/blob/7f61186cbcbc43cd6ccd717abd097f638b786984/sklearn/tree/tests/test_tree.py#L504-L547

and remove:

https://github.com/scikit-learn/scikit-learn/blob/7f61186cbcbc43cd6ccd717abd097f638b786984/sklearn/tree/tests/test_tree.py#L548-L568